Add acct_nt (T2 Lock) (#463)
comcode-org/hackmud_wiki: acct_nt.mdx

</details>

### Mode: large deposit

<details>
<summary>Spoilers for acct_nt's solutions </summary>

Find the deposit that acct_nt asks for and provide it as the acct_nt argument. It may be a bit off from the timestamp.

</details>

### Mode: net

<details>
<summary>Spoilers for acct_nt's solutions </summary>

Add up all the transactions between the two timestamps, with incoming transactions being positive and outgoing transactions being negative. The actual starting and ending timestamps may be incorrect by up to one minute.

</details>

### Mode: total earned

<details>
<summary>Spoilers for acct_nt's solutions </summary>

Add up all the incoming transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.

</details>

### Mode: total spent

<details>
<summary>Spoilers for acct_nt's solutions </summary>

> Add up all the outgoing transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.

Author: Are you sure they use net? That just seems like a redundant thing then, and it should probably just be put under one category.

```

If this check passes instead, the lock will return the following info:

```
>>>user.loc {acct_nt:""}
What was the net GC between 240423.1042 and 240425.2107
```

## Solving

### Example unlock

<details>
  <summary>Spoilers for acct_nt's solutions</summary>

```
>>user.loc {acct_nt:"2B520MGC"}
WARNING: BINMAT security shell inactive. Intelligent defense system offline.
LOCK_UNLOCKED
System beta breached.
Connection terminated.
```

</details>

### acct_nt

<details>
<summary>Spoilers for acct_nt's solutions </summary>

Depending on the mode acct_nt is in, "large deposit", "net", "total earned/total spent (with/without memos)", a different approach may be required. A check of [[accts.transactions]] is needed to find the correct answer. Since acct_nt is only minute precise, it may be needed to add/subtract transaction amounts in order to get a correct solve.

</details>

### Mode: large deposit

<details>
<summary>Spoilers for acct_nt's solutions </summary>

Find the deposit that acct_nt asks for and provide it as the acct_nt argument. It may be a bit off from the timestamp.

</details>

### Mode: net

<details>
<summary>Spoilers for acct_nt's solutions </summary>

> Look at the timestamps that acct_nt gives you, and add up all the transactions from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.

Reviewer: All articles should be written in third person with no referential nouns (e.g. you, your), except for Tutorials, FAQs, and Writeups.

```suggestion
Add up all the transactions between the two timestamps, with incoming transactions being positive and outgoing transactions being negative. The actual starting and ending timestamps may be incorrect by up to one minute.
```

</details>

### Mode: total earned

<details>
<summary>Spoilers for acct_nt's solutions </summary>

> Look at the timestamps that acct_nt gives you, and add up all the transactions where you gained money from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.

Reviewer: ```suggestion
Add up all the incoming transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
```

</details>

### Mode: total spent

<details>
<summary>Spoilers for acct_nt's solutions </summary>

> Look at the timestamps that acct_nt gives you, and add up all the transactions where you lost money from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.

Reviewer: ```suggestion
Add up all the outgoing transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
```

</details>

### Mode: with/without memos

<details>
<summary>Spoilers for acct_nt's solutions </summary>

> With or without memos indicates whether or not if you should consider transactions with memos. If it says with memos, make sure the transactions have memos before adding them, otherwise ignore them.

Reviewer: ```suggestion
Indicates whether only transactions with (or without) memos should be considered.
```

> Memos are a little message that can be sent with a transaction. It displays as memo: "message here" in [[accts.transactions]].

Reviewer: I don't think that memos need to be explained here; if needed, they can be explained on the accts.transactions page.
```suggestion
```
